### DISCUSSION TITLE
Starting my first DKron server

### DISCUSSION THREAD

**Discussion by arnaudcornely** [dkron-io/dkron#1193] [Q&A] [answered]
Hi, I am playing with the idea of running DKron at scale in my environment but I am stuck at the very first step.
I am just trying to run DKron from on single windows PC, to figure if this can work fo rme.

The agent start, well, or so it seems, but and attempt to create a new job fails with a simple "Bad request" response from the UI.
I see the POST was received by the agent, but there is not a single place helping me to figure what I do wrong.

How do I go past this?

This is my startup command:
>.\dkron.exe agent --server  --log-level debug --bootstrap-expect 1

**Comment by yvanoers** [comment, marked as answer]
I suggest posting directly to the API. That might give you a better error message to work with.
If that doesn't help, post the payload here and we can have a look at it.

**Comment by Szymok** [comment]
Could someone provide example of configuration a job in dkron ui? I want to know what specificaly i need to fulfill

**Comment by yvanoers** [comment]
![2023-06-02_21-59](https://github.com/distribworks/dkron/assets/4980327/c284470e-3dba-4bf5-84ea-1ca5c9f7ce9e)
